apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -380,10 +380,20 @@ defmodule Aecore.Chain.Worker do
 
     accounts_unique =
       block.txs
-      |> Enum.map(&[&1.data.sender, &1.data.payload.receiver])
+      |> Enum.map(fn tx ->
+        case tx.data.type do
```

Comment:
aren't we back at the patternmatch that can't work?

Reply:
You are correct, idk where my thinking was. Fixing it now